src/qt/bitcoingui.cpp

```diff
@@ -1452,16 +1452,17 @@ void BitcoinGUI::updateScraperIcon(int scraperEventtype, int status)
 
     int64_t nConvergenceTime = ConvergedScraperStatsCache.nTime;
 
-    std::string sExcludedProjects = {};
+    std::string sExcludedProjects;
+    std::string sIncludedScrapers;
+    std::string sExcludedScrapers;
+    std::string sScrapersNotPublishing;
 
-    bool bExcludedProjects = false;
+    bool bDisplayScrapers = false;
 
-    // If the convergence cache has excluded projects...
-    if (ConvergedScraperStatsCache.vExcludedProjects.size())
+    // If the convergence cache has excluded projects and/or is in fallback (by parts)...
+    if (ConvergedScraperStatsCache.Convergence.vExcludedProjects.size() || ConvergedScraperStatsCache.Convergence.bByParts)
```

Comment:
Is `vExcludedProjects` ever empty when `bByParts == true`? Not sure if we need the check because we're looping over it right below.

Reply:
Actually, it seems like we can drop the `bByParts` flag--this looks like the only spot we're using it.

Reply:
You're right the second check in the if can be removed because it is redundant, because I changed the logic a whileback to always drop to by Parts convergence if there is a missing project when the manifest level convergence is tried. (This was that commented out stuff that I removed this time around.)

I still want to retain the bByParts flag for future use in the rpc function.